jvb/src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -444,4 +457,11 @@ public void removeReceiver(String receiverId)
             receiverVideoConstraintsChanged(receiverVideoConstraintsMap.values());
         }
     }
+
+    interface EventHandler {
```

Comment:
Can this be in `Endpoint` instead?

Reply:
Why there instead of here?  Then everywhere that wants to use it has to care about what the real type of an `AbstractEndpoint` is.  It's ok for subclasses not to fire some events (e.g. `OctoEndpoint` never firing `iceSucceeded`)